Apps/Src/SendTritium.c

```diff
@@ -350,8 +380,9 @@ static void readInputs()
 static uint8_t map(uint8_t input, uint8_t in_min, uint8_t in_max, uint8_t out_min, uint8_t out_max)
```

Comment:
FYI that a PR is about to merged in to make this map function return a float value. Will probably need to be merged into this branch but shouldn't have any conflicts I think though.

Reply:
Actually now that I look a little further down, it might take some merge resolution where you are using the map function.